src/app/pages/dashboard/components/practice-test-report-card/practice-test-report-card.html

```diff
@@ -1,12 +1,12 @@
 <button clear (click)="showDrivingFaultModal()">
-  <ion-card class="practice-mode-card">
+  <ion-card class="practice-mode-card" id="practice-mode-card">
     <ion-grid>
       <ion-row>
         <ion-col col-36 col-lg-26 align-self-center>
           <img src="assets/imgs/dashboard/practice-mode.png" alt="Practice a test report" class="icon-practice-test"/>
         </ion-col>
         <ion-col col-58 col-lg-44 align-self-center class="button-col">
-          <ion-button class="mes-neutral-button">
+          <ion-button class="mes-neutral-button" id="practice-mode">
```

Comment:
same again, call this "practice-mode-button"

Reply:
done